VulkanHppGenerator.cpp

```diff
@@ -9102,7 +9102,9 @@ std::string
     if ( !handle.second.secondLevelCommands.empty() )
     {
       assert( !handle.second.constructorIts.empty() );
+#if !defined( NDEBUG )
       auto constructorCommandIt = m_commands.find( handle.second.constructorIts.front()->first );
+#endif
```

Comment:
could also wrap this endif around the asserts.

Reply:
I think, it's fine this way.